pkg/memory-monitor/memory-monitor-ctr-init.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# We need extra wrapper to run the memory monitor and "sleep" in the background forever,
+# because the memory monitor is a demonized process and the containerd will decide that the
+# container is dead if the process is not running in the foreground.
+
+# Run the demonized memory monitor
+/sbin/memory-monitor
+
+# Let this script run forever, while the memory monitor is running
+while pgrep "/sbin/memory-monitor" > /dev/null ; do
+    sleep 30
+done
```

Comment:
Isn't this what `waitpid` is for?

Reply:
Hmm... I'll check. Did not know about the utility.

Reply:
I've removed the entire script